security/wordpress/rename-admin-user.php

```diff
@@ -93,9 +93,9 @@ function rsssl_rename_admin_user() {
 
 
 function rsssl_maybe_rename_admin_user() {
-	rsssl_do_fix('rsssl_rename_admin_user');
+//	rsssl_do_fix('rsssl_rename_admin_user');
```

Comment:
Waarom is dit uitgecommentaard?

Reply:
Deze fix zit nu op een admin_init hook, de rsssl_do_fix geeft nog fatal errors. Lijkt me beter om die functie in 1x te fixen als alle geupdate integraties in een branch zitten

Reply:
Ahh duidelijk. Prima.